src/python/graplctl/graplctl/aws/commands.py

```diff
@@ -18,7 +18,7 @@ def aws(
     ctx: click.Context,
 ) -> None:
     """commands for managing grapl aws resources"""
-    pass
+    pass  # noqa
```

Comment:
I personally like the explicit pass, but I'm open to having autoflake remove this automatically since strictly speaking its unnecessary.

Reply:
I also prefer the explicit `pass`. I use it like `todo!()` in rust.